app/Repositories/TaskRepositories.php

```diff
@@ -15,12 +15,14 @@ class TaskRepositories implements TaskRepositoryInterface
 {
     public function index(PaginationDTO $paginationDTO, int $todo_id): Collection
     {
+        Todo::findOrFail($todo_id);
```

Comment:
You already have a show method that does the same thing. For a variety of tests, check if there is a todo entry in the action class.